docs/docusaurus/docs/reference/learn/data_quality_use_cases/missingness.md

```diff
@@ -1,8 +1,153 @@
 ---
 sidebar_label: 'Missingness'
-title: 'Missingness'
+title: 'Manage missing data with GX'
 ---
 
-import InProgress from '../../../core/_core_components/_in_progress.md'
+If you've ever struggled with missing data messing up your reports or skewing your machine learning models, this guide will show you how to leverage Great Expectations (GX) to keep your data clean and reliable.
 
-<InProgress/>
\ No newline at end of file
+**Missing data**, often referred to as **missingness**, denotes the absence of expected information in a dataset. These gaps can be represented by `NULL` values in SQL databases or manifest differently across systems, impacting data quality. Missing data poses challenges such as disrupting calculations, skewing analyses, and compromising machine learning models. Given the critical role of data quality, managing missing data effectively is essential to ensure accurate, reliable, and actionable datasets.
+
+Managing missing data involves:
+
+1. Identifying where data is missing
+2. Understanding the causes of the missing data
+3. Deciding on appropriate remedial actions
+
+GX is an open-source data quality tool that provides powerful features for managing and validating data integrity, including handling missing data. Supported by an active community, GX allows you to create customizable checks, known as Expectations, that help maintain high data quality standards across diverse workflows.
+
+In this guide, we will walk you through utilizing GX to proficiently handle missing data, ensuring your datasets remain clean, consistent, and ready for analysis or machine learning applications. You will learn step-by-step methods for writing Expectations, setting up automated checks, and making informed decisions about managing various types of missing data.
+
+## Prerequisite knowledge
+
+Before diving into this guide, ensure you have a basic understanding of GX components and workflows. If you're new to GX, we recommend starting with our [GX Overview](/core/introduction/gx_overview.md) to familiarize yourself with key concepts and setup procedures.
+
+## Data preview
+
+Below is a sample of the dataset referenced throughout this guide:
+
+| type     | sender_account_number  | recipient_fullname | transfer_amount | transfer_date       | errors |
+|----------|------------------------|--------------------|-----------------|---------------------|--------|
+| domestic | 244084670977           | Jaxson Duke        | 9143.40         | 2024-05-01 01:12    | NULL   |
+| NULL     | 123456789012           | Jane Smith         | 5000.00         | NULL                | NULL   |
+
+This dataset includes potential for missing data in `type` and `transfer_date` columns.
```

Comment:
I think it would be helpful to explain the inclusion of the `errors` column in the dataset as well, just so that readers understand it's populated if there is an error with the record.